projects/plugins/jetpack/_inc/client/recommendations/summary/style.scss

```diff
@@ -14,7 +14,7 @@
 	box-shadow: 0px 2px 6px rgba( 0, 0, 0, 0.03 ), 0px 1px 2px rgba( 0, 0, 0, 0.03 );
 	border: 1px solid $gray-lighten-20;
 
-	@include breakpoint( '<660px' ) {
+	@include breakpoint( '<782px' ) {
```

Comment:
Layout was somewhat broken between 660 and 782 so I updated it to switch to the column view earlier